packages/fretboard/PKGBUILD

```diff
@@ -0,0 +1,31 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Florian Hülsmann <fh@cbix.de>
+
+pkgname=fretboard
+pkgver=7.0
+pkgrel=1
+pkgdesc='Simple guitar chord lookup tool'
+arch=(aarch64 x86_64)
+url='https://apps.gnome.org/Fretboard/'
+license=(GPL-3.0-or-later)
+groups=(pro-audio)
```

Comment:
Not sure if it makes sense to add `gnome-extra` since it's featured on https://apps.gnome.org/